playbooks/ansible-deployment-example.yml

```diff
@@ -0,0 +1,104 @@
+---
+- name: get awx-operator git repo
+  ansible.builtin.git:
+    repo: "https://github.com/ansible/awx-operator.git"
+    dest: "~/awx-operator"
+    version: "{{ awx_operator_version }}"
+    force: true
+
+- name: deploy kustomization file
+  copy:
+    content: |
+      ---
+      apiVersion: kustomize.config.k8s.io/v1beta1
+      kind: Kustomization
+      resources:
+        # Find the latest tag here: https://github.com/ansible/awx-operator/releases
+        - github.com/ansible/awx-operator/config/default?ref={{ awx_operator_version }}
```

Comment:
These lines download the AWX Operator directly, so it's redundant to download with Git.

Reply:
Removed the git checkout part, thanks!